includes/class-wc-payments-account.php

```diff
@@ -302,6 +302,66 @@ public function get_branding_secondary_color() : string {
 		return isset( $account['branding']['secondary_color'] ) ? $account['branding']['secondary_color'] : '';
 	}
 
+	/**
+	 * Gets the deposit schedule interval.
+	 *
+	 * @return string interval e.g. weekly, monthly.
+	 */
+	public function get_deposit_schedule_interval(): string {
+		$account = $this->get_cached_account_data();
+		return $account['deposits']['interval'] ?? '';
+	}
+
+	/**
+	 * Gets the deposit schedule weekly anchor.
+	 *
+	 * @return string weekly anchor e.g. monday, tuesday.
+	 */
+	public function get_deposit_schedule_weekly_anchor(): string {
+		$account = $this->get_cached_account_data();
+		return $account['deposits']['weekly_anchor'] ?? '';
+	}
+
+	/**
+	 * Gets the deposit schedule monthly anchor.
+	 *
+	 * @return string monthly anchor e.g. 1, 2.
+	 */
+	public function get_deposit_schedule_monthly_anchor(): string {
+		$account = $this->get_cached_account_data();
+		return $account['deposits']['monthly_anchor'] ?? '';
+	}
+
+	/**
+	 * Gets the number of days payments are delayed for.
+	 *
+	 * @return string  e.g. 2, 7.
+	 */
+	public function get_deposit_delay_days(): string {
+		$account = $this->get_cached_account_data();
+		return $account['deposits']['delay_days'] ?? '';
```

Comment:
Should we use a fallback value of `7` rather than an empty string?

Reply:
I've updated this to return an `int|null`. Just to limit the use of magic numbers.
Will use the magic number in two places
https://github.com/Automattic/woocommerce-payments/blob/52f17d2abf40c700c2885ba4cf55bd67af8a8b2e/includes/class-wc-payment-gateway-wcpay.php#L2262
and 
https://github.com/Automattic/woocommerce-payments/blob/52f17d2abf40c700c2885ba4cf55bd67af8a8b2e/client/data/settings/selectors.js#L107